libs/deps/mas/merch-card.js

```diff
@@ -1673,7 +1673,7 @@ merch-card[variant="ccd-slice"] [slot='image'] img {
     /* colors */
     --merch-card-background-color: var(--spectrum-gray-background-color-default, #fff);
     --consonant-merch-card-border-color: #eaeaea;
-    --color-accent: #1473E6;
+    --color-accent: rgb(59, 99, 251);
```

Comment:
we should remove this line, it is not used.

Reply:
its generated file, this one comes from global.css.js

Reply:
discussed with @yesil and indeed we could remove it, but i didn't manage to address it before it got merged, so will do it in the upcoming PR